test/fixtures/dialects/bigquery/bigquery_parameters.yml

```diff
@@ -88,14 +88,12 @@ file:
             function_name:
               function_name_identifier: parse_date
             bracketed:
-            - start_bracket: (
-            - expression:
+              start_bracket: (
+              expression:
                 literal: '"%Y%m"'
-            - comma: ','
-            - expression:
-                column_reference:
-                  identifier: year
-            - end_bracket: )
+              comma: ','
+              date_part: year
```

Comment:
This looks like an improvement in parsing (correctness).

Reply:
Yeah, it is 👍